core-web/libs/block-editor/src/lib/components/dot-block-editor/dot-block-editor.component.ts

```diff
@@ -361,6 +362,13 @@ export class DotBlockEditorComponent implements OnInit, OnDestroy {
                 allowedBlocks: this._allowedBlocks
             }),
             Placeholder.configure({ placeholder: this.placeholder }),
+            Youtube.configure({
+                height: 400,
+                width: 500,
```

Comment:
Let's make this width: 400 x height: 300 because in the edit contentlet we don't have too much space.